### PR TITLE
Remove jQuery, optimize JS

### PR DESCRIPTION
**Pull request by gregurco**
| Q                | A
| ---------------- | -----
| Bug fix          | yes
| New feature      | no
| BC breaks        | no
| Deprecations     | no
| Tests pass       | yes
| Fixed tickets    | [#188]
| License          | MIT

What I did:
- remove jQuery - I don't think we need the whole library for 1-2 methods that can be written on clean JS + I think issue in #188 was cause because conflict between different jQuery versions. It's better to remove it at all than to fix conflicts.
- moved script tab out of the loop - there was script tab in look and bloc was generated for each request. Now there are only one script tag and js code per whole page
- removed id if there are multiple elements with same id. It's not allowed in html to have multiple elements with same id. Technically it can work, but its better to avoid such as cases.
- remove variable that tracks first element in look. Used `loop.index0` for that

@florianpreusner what do you think? It's not finished and tested good yet but I want to know your opinion. Finally we can move back jQuery anytime if we will observe that some functionality that strictly depends on it appears.

### CODE REVIEW

File: src/Resources/public/css/main.css [hunk 1]
```diff
@@ -163,8 +163,10 @@
     margin: 16px 0 10px 0;
 }
 
-
-
 #guzzle_profiler .content {
     padding: 10px;
 }
+
+.content.hide {
```
Comment (florianpreusner) [inline src/Resources/public/css/main.css]: why not adding #guzzle_profiler in front? just to avoid conflicts with other css styles...

Comment (gregurco) [reply src/Resources/public/css/main.css]: yep, I thought about that and it's next that I planned to do :slightly_smiling_face:

Comment (gregurco) [reply src/Resources/public/css/main.css]: Now the main question is about js part

**Comment by florianpreusner** [comment]
- remove jQuery: 👍  nice idea. you are right, we only use 0,0001% of this library!
- moved script tab out of the loop: 👍 
- removed id if there are multiple elements with same id: ooooops 😆 👍 
- Used loop.index0: nice! much cleaner!

**Comment by gregurco** [comment]
@florianpreusner thank you for feedback :slightly_smiling_face: I asked for help with testing in #188 but I will test it too

**Comment by gregurco** [comment]
@florianpreusner please take a look :) I renamed and normalized all classes to prevent conflicts. I used BOM CSS naming convention. And also I tested and it seems to be ok :slightly_smiling_face:

**Comment by florianpreusner** [comment]
@gregurco Can you commit a small change to trigger SensioLabsInsight again? I reactivated this check. When there is inactivity the automatic check gets disabled :/

**Comment by gregurco** [comment]
@florianpreusner triggered, verified and merged :slightly_smiling_face:

**Comment by florianpreusner** [comment]
@gregurco, Good job! Thanks!

**Pull request merged**